src/OrchardCore.Modules/OrchardCore.Deployment/Views/DeploymentPlan/Display.cshtml

```diff
@@ -112,21 +116,30 @@
     </div>
 </div>
 <script at="Foot" type="text/javascript">
+    function replaceDiacritics(str) {
+        return str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
+    }
+
     $(function () {
         var searchBox = $('#search-box');
 
         // On each keypress filter the list of cards
         searchBox.keyup(function (e) {
-            var search = $(this).val().toLowerCase();
+            var search = replaceDiacritics($(this).val().toLowerCase());
             if (search == '') {
                 searchBox.val('');
-                $('#cards > div').toggle(true);
+                $('.card-item').attr('style', 'display:flex !important');
             }
             else {
-                $('#cards > div').each(function () {
-                    var filter = $(this).children().first().html().toLowerCase();
+                $('.card-item').each(function () {
+                    var filter = replaceDiacritics($(this).children().children().first().html().toLowerCase());
```

Comment:
Should you replace this selector with something more specific instead of calling .children().children() ? Maybe $(this).find(".card-title") ?

Reply:
I believe that we are searching in the entire card including it's "body" -> description